providers/ns1/ns1Provider.go

```diff
@@ -284,33 +284,33 @@ func convert(zr *dns.ZoneRecord, domain string) ([]*models.RecordConfig, error)
 			Original: zr,
 		}
 		rec.SetLabelFromFQDN(zr.Domain, domain)
-		switch rtype := zr.Type; rtype {
+		rtype := zr.Type
+		var err error
+		switch rtype {
 		case "DNSKEY", "RRSIG":
 			// if a zone is enabled for DNSSEC, NS1 autoconfigures DNSKEY & RRSIG records.
 			// these entries are not modifiable via the API though, so we have to ignore them while converting.
 			// 	ie. API returns "405 Operation on DNSSEC record is not allowed" on such operations
 			continue
 		case "ALIAS":
 			rec.Type = rtype
-			if err := rec.SetTarget(ans); err != nil {
-				return nil, fmt.Errorf("unparsable %s record received from ns1: %w", rtype, err)
-			}
+			err = rec.SetTarget(ans)
 		case "URLFWD":
 			rec.Type = rtype
-			if err := rec.SetTarget(ans); err != nil {
-				return nil, fmt.Errorf("unparsable %s record received from ns1: %w", rtype, err)
-			}
+			err = rec.SetTarget(ans)
 		case "CAA":
 			//dnscontrol expects quotes around multivalue CAA entries, API doesn't add them
 			xAns := strings.SplitN(ans, " ", 3)
-			if err := rec.SetTargetCAAStrings(xAns[0], xAns[1], xAns[2]); err != nil {
-				return nil, fmt.Errorf("unparsable %s record received from ns1: %w", rtype, err)
-			}
+			err = rec.SetTargetCAAStrings(xAns[0], xAns[1], xAns[2])
+		case "TXT":
+			err = rec.SetTargetTXTQuotedFields(ans)
```

Comment:
```suggestion
			err = rec.SetTargetTXT(ans)
```

ns1's api always adds quotes around strings. Adding them ourselves results to extraneous
quotes finding their way into the record.
ie.
       v := "asdf"
       SetTargetTXT(v) becomes "asdf" on the wire and a record value of `asdf`
       SetTargetTXTQuotedFields(v) becomes "\"asdf\"" on the wire and a record value of `"asdf"`
... which breaks integration on "asdf" != "\"asdf\""


with this fix, LGTM and all tests, manual and integration are succesful!